book/src/user/install.md

```diff
@@ -1,13 +1,35 @@
 # Installing Zebra
 
-Zebra is still under development, so there is no supported packaging or
-install mechanism. To run Zebra, check out the git repository:
-```
-git clone https://github.com/ZcashFoundation/zebra
-```
-and then run
-```
-cargo build
-```
-
-Be aware that Zebra is still in an extremely early stage of development.
\ No newline at end of file
+`zebrad` is still under development, so there is no supported packaging or
+install mechanism. To run `zebrad`, follow the instructions to compile `zebrad`
+for your platform:
+
+1. Install [`cargo` and `rustc`](https://www.rust-lang.org/tools/install).
+     - Using `rustup` installs the stable Rust toolchain, which `zebrad` targets.
+2. Install Zebra's build dependencies:
+     - **libclang:** the `libclang`, `libclang-dev`, `llvm`, or `llvm-dev` packages, depending on your package manager
+     - **clang** or another C++ compiler: `g++`, `Xcode`, or `MSVC`
+3. Run `cargo install --locked --git https://github.com/ZcashFoundation/zebra --tag v1.0.0-alpha.19 zebrad`
```

Comment:
Notice we have a new spot to update versions here.

Reply:
Thanks! I noted here https://github.com/ZcashFoundation/zebra/pull/2973/files#r739447452 so I don't forget